tools/firmware-utils/src/tplink-safeloader.c

```diff
@@ -2280,14 +2332,17 @@ static inline uint8_t bcd(uint8_t v) {
 
 
 /** Generates the soft-version partition */
-static struct image_partition_entry make_soft_version(struct device_info *info, uint32_t rev) {
-	size_t part_len = sizeof(struct soft_version);
-	if (info->soft_ver_compat_level > 0)
-		part_len += sizeof(uint32_t);
-
-	struct image_partition_entry entry =
-	    alloc_image_partition("soft-version", part_len+1);
-	struct soft_version *s = (struct soft_version *)entry.data;
+static struct image_partition_entry make_soft_version(
+	const struct device_info *info, uint32_t rev
+) {
```

Comment:
can you move the ')' on the same line of the arguments.
The file already mixes different styles, but let's not introduce another style ;).